src/common/storage/src/config.rs

```diff
@@ -46,7 +46,7 @@ use serde::Serialize;
 pub struct StorageConfig {
     pub num_cpus: u64,
     pub allow_insecure: bool,
-
+    pub max_storage_io_requests: Option<u64>,
```

Comment:
it seems StorageConfig mostly concerns about the connection locations of different storage providers, maybe max_storage_io_requests can also be considered as a QueryConfig ?